app/controllers/idv/review_controller.rb

```diff
@@ -63,7 +75,7 @@ def idv_params
     end
 
     def phone_confirmation_required?
-      !idv_params[:phone_confirmed_at] || idv_params[:phone] != current_user.phone
+      idv_params[:phone] != current_user.phone
```

Comment:
hmm. maybe we don't have proper test coverage around this, but I believe the `or` condition is necessary for re-entry of the idv flow.